src/main/java/net/minecraftforge/event/entity/living/LivingUpdateBlockEvent.java

```diff
@@ -0,0 +1,51 @@
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.common.ForgeHooks;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+
+/**
+ * LivingUpdateBlockEvent is fired whenever a living Entity (excluding player) changes or replaces block state. <br>
+ * <br>
+ * Entities which can trigger this event: Sheep, EnderDragon, Enderman, Rabbit, Silverfish, Wither, Zombie
+ * <br>
+ * This event is fired via the {@link ForgeHooks#onLivingUpdateBlock(EntityLivingBase, BlockPos, IBlockState)}.<br>
```

Comment:
no, put where in each entity the Forgehooks method is called.
Saying it's fired from ForgeHooks is useless, we want to know where in each entity it fires